src/tests/cli-tests/src/contracts/yul/contract.yul

```diff
@@ -0,0 +1,80 @@
+//! { "cases": [ {
```

Comment:
Please remove this compiler-tester metadata.

Reply:
done